File: Comparative_Analysis_Exercise/C/Student_Reg_Status_Checker.c
// Student Registration Status Checker in C
#include <stdio.h>


int main() {
    char stu_name[30];
    int units;

    printf("Enter your name: ");
    scanf("%s", stu_name);
    printf("Enter no. of units registered: ");
    scanf("%d", &units);

    if (units > 7) {
        printf("---------------------------------------\n");
        printf("Student Name: %s\n", stu_name);
        printf("Registered Units: %d\n", units);
        printf("Status: Overload - Approval Required.\n");
        printf("---------------------------------------\n");
    } else {
        printf("---------------------------------------\n");
        printf("Student Name: %s\n", stu_name);
        printf("Registered Units: %d\n", units);
        printf("Status: Registration Accepted.\n");
        printf("---------------------------------------\n");
    }

    return 0;

}

File: Comparative_Analysis_Exercise/Java/StudentRegStatusChecker.java
// Student Registration Status Checker in Java
import java.util.Scanner;

public class StudentRegStatusChecker {
    public static void main(String[] args) {
        Scanner student = new Scanner(System.in);
        System.out.print("Enter you name: ");
        String stu_name = student.nextLine();
        System.out.print("Enter no. of units registered: ");
        int units = student.nextInt();

        String status = (units > 7) ? "Overload - Approval Required." : "Registration Accepted.";
        
        System.out.println("---------------------------------------");
        System.out.println("Student Name: " + stu_name);
        System.out.println("Units Registered: " + units);
        System.out.println("Status: " + status);
        System.out.println("---------------------------------------");

        student.close();
        
    }
}